lib/sinks/sqlInjection/index.js

```diff
@@ -3,3 +3,9 @@
 module.exports = {};
 module.exports['sequelize.prototype.query'] = require('./sequelize');
 module.exports['mysql/lib/Connection.query'] = require('./mysql');
+module.exports[
+  'pg.Connection.prototype.query (String)'
+] = require('./pg_string');
```

Comment:
Can we nest `pg`-related in a `pg` folder?

i.e.
```suggestion
] = require('./pg/string');
```

Reply:
👍